dcicutils/cloudformation_utils.py

```diff
@@ -232,6 +232,35 @@ def find_stack_resource(self, stack_name_token, resource_logical_id, attr=None,
                     return getattr(summary, attr, default)
         return default
 
+    @classmethod
+    def find_lambda_function_names(cls, name_pattern) -> list:
```

Comment:
You might want to test this, I would just do something simple like mock out a response from `list_functions` and verify it gives the functions you were looking for.

Reply:
I think there is support for a lambda client where you specify the names  you want to test against, so that should be easy.